pkg/controller/nodes/task/transformer.go

```diff
@@ -154,6 +156,15 @@ func ToTaskExecutionEvent(input ToTaskExecutionEventInputs) (*event.TaskExecutio
 	} else {
 		tev.Metadata.InstanceClass = event.TaskExecutionMetadata_DEFAULT
 	}
+	if input.EventConfig.RawOutputPolicy == config.RawOutputPolicyInline {
+		tev.InputValue = &event.TaskExecutionEvent_InputData{
```

Comment:
Since this requires a new Admin version, what do we need to change to make it backward compatible?

Reply:
the over the wire field number is the same for input URIs. The only change is for deployments using the `RawOutputPolicyInline` for which input values are already not being passed